docs/index.rst

```diff
@@ -0,0 +1,70 @@
+Bayesian Tracker (btrack) 🔬💻's
+================================
+
+|logo|
+
+BayesianTracker (``btrack``) is a Python library for multi object tracking, used to reconstruct trajectories in crowded fields.
```

Comment:
is this meant to have ```` (i.e. twice)

Reply:
Yes, [reStructuredText works that way](https://sphinx-tutorial.readthedocs.io/cheatsheet/#rst-cheat-sheet)